pkg/kapp/crdupgradesafety/change_validator.go

```diff
@@ -71,6 +71,48 @@ func EnumChangeValidation(diff FieldDiff) (bool, error) {
 	return handled(), nil
 }
 
+// RequiredFieldChangeValidation adds a validation check to ensure that
+// existing required fields can be marked as optional in a CRD schema:
+// - No new values are added as required that did not previously have
+// any required fields present
+// - Existing values are removed from the required field
```

Comment:
nit: Maybe? 
```suggestion
// - Existing values are kept/removed from the required field
```
Or `can be` instead of `are`?